rbd/system/cli/rbd_cli_import_export_diff.py

```diff
@@ -42,6 +42,12 @@ def exec_cmd(args):
     if k_m:
         combinations = cli.generate_combinations('image_size', 'object_size',
                                                  'image_format', 'data_pool')
+        if cli.ceph_version == 3:
```

Comment:
I think the best way to check is to use starts_with. if ceph_version is 3.3 this condition might fail. Can you check what is the value it returns?

Reply:
@rakeshgm i tested with 3.3 build and it worked without any issue.